kotest-bom/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+   `java-platform`
+   `maven-publish`
+}
+
+version = Ci.publishVersion
+
+val bomProject = project
+val excludeFromBom = listOf("kotest-examples", "kotest-tests")
+fun projectsFilter(candidateProject: Project) =
+   excludeFromBom.all { !candidateProject.name.contains(it) }
+      && candidateProject.name != bomProject.name
+
+rootProject.subprojects.filter(::projectsFilter).forEach { bomProject.evaluationDependsOn(it.path) }
+
+dependencies {
+   constraints {
+      rootProject.subprojects.filter { project ->
+         project.tasks.findByName("publish")?.enabled == true &&
+            projectsFilter(project)
+      }.forEach { api(project(it.path)) }
+   }
+}
+
+publishing {
+   publications {
+      create<MavenPublication>("Bom") {
```

Comment:
What is "Bom" here?

Reply:
It's just an arbitrary name for the publication which doesn't propagate to anything published as far as I can tell. I'll change it to KotestBom for completeness.